landa/organization_management/doctype/landa_member/landa_member.py

```diff
@@ -106,6 +106,28 @@ def get_full_name(first_name, last_name):
 	return (first_name or "") + (" " if (last_name and first_name) else "") + (last_name or "")
 
 
+@frappe.whitelist()
+def clear_special_yearly_fishing_permits(members):
+	"""Remove all checkboxes in "Erlaubnisscheine Gewässerfonds" for the given members."""
+	members = frappe.parse_json(members)
+
+	permit_fields = [
+		"has_special_yearly_fishing_permit_1",
+		"has_special_yearly_fishing_permit_2",
+		"has_special_yearly_fishing_permit_3",
+		"has_special_yearly_fishing_permit_4",
+		"has_special_yearly_fishing_permit_5",
+		"has_special_yearly_fishing_permit_6",
+		"has_special_yearly_fishing_permit_7",
+	]
+
+	for member in members:
```

Comment:
We cannot be certain that `member` is a string here. Better enforce it, to be safe.